go/common/headers.go

```diff
@@ -20,29 +20,33 @@ var hasherPool = sync.Pool{
 // Header is a public / plaintext struct that holds common properties of rollups batches.
 // Making changes to this struct will require GRPC + GRPC Converters regen
 type Header struct {
```

Comment:
I reordered the fields here, to first have the ones in Geth's `types/Header`, then the custom fields. This makes it easier to understand what we're adding to the Geth type.